rad/examples/counter.rad

```diff
@@ -0,0 +1,64 @@
+;; A “counter” chain that exposes an `increment` mutator and a `get-value`
+;; accessor.
+;;
+;; This file provides functions to interact with a counter chain.
+;;
+;;     (counter/init-chain! "my-chain-id") ;; => :ok
+;;     (counter/get-value "my-chain-id") ;; => 0
+;;     (counter/increment! "my-chain-id") ;; => 1
+;;     (counter/get-value "my-chain-id") ;; => 1
+;;
+
+(load! "rad/prelude.rad")
+
+(import prelude/test '[assert-equal] :unqualified)
+
+(def counter/prelude
```

Comment:
I think now that we have modules and qualified imports this doesn't need the leading `counter/`?

Reply:
Removing the `counter/` prefix would indeed make sense if this file was a module. But making this file a module would require more boilerplate code or another wrapper in the bash script. This is why I opted for this file to be just a script.